src/Application/Authorisation/Suggest/View.elm

```diff
@@ -84,6 +89,16 @@ view context model =
                 , Html.text "?"
                 ]
 
+          else if hasRawResources then
+            Html.div
+                [ T.mt_10 ]
```

Comment:
This is more an error on my part, but we should put `Html.div [ T.mt_10 ]` a level higher, so that the `if` condition only applies to the child nodes of this element.

Reply:
Yep, makes sense. I'll move it up.